<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosInfoSetPage</class>
 <widget class="QWidget" name="PosInfoSetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="skinWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#skinWidget{
	background-color:#ffffff;
}</string>
     </property>
     <widget class="QLabel" name="smartLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>290</y>
        <width>94</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>智能版</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="skinSetLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>234</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>皮肤设置：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="flatSkinLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>94</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>扁平版</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="balckSkinLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>94</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>黑色版</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="SKinSetCellButton" name="balckSkinButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>94</width>
        <height>54</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="SKinSetCellButton" name="flatSkinButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>94</width>
        <height>54</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="SKinSetCellButton" name="smartButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>94</width>
        <height>54</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="posModelLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>194</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>POS机型号：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="posCodeLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>154</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>28888824656</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>154</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>POS机编号：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>114</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>门店名称：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="shopNameLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>114</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>高新原店</string>
      </property>
     </widget>
     <widget class="QLabel" name="shopCodeLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>74</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>23355568</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>74</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>门店编号：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="curVersionLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>34</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>POS-1.2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>34</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>102</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>当前版本：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="posTestButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-image: url(:/kPosSetImage/buttonNormal.png);
color:#ffffff;
}

QPushButton:pressed{
border-image: url(:/kPosSetImage/buttonPressed.png);
}</string>
      </property>
      <property name="text">
       <string>测试</string>
      </property>
     </widget>
     <widget class="KPosComboBox" name="posModelComBox" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>190</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SKinSetCellButton</class>
   <extends>QPushButton</extends>
   <header>SKinSetCellButton.h</header>
  </customwidget>
  <customwidget>
   <class>KPosComboBox</class>
   <extends>QWidget</extends>
   <header>KPosComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
